### DISCUSSION TITLE
[Feature Request] show what portals instances are in and give us the option to add and remove them from portals

### DISCUSSION THREAD

**Discussion by Whurrhurr** [diesel-modding/BeardLib-Editor#617] [Ideas]
self explanatory. it's annoying to move an instance in a cloned vanilla heist only to find out "oh oops it's in a portal and you had no way of knowing about this beforehand"

**Comment by Luffyyy** [comment]
Uh I'm not sure how portals can have instances inside of them. I did patch a thing that would auto fill instance units to portals. Now, in theory it *should* work but it can become quite complicated to deal with instance units.
